### DISCUSSION TITLE
Endre på DatePicker og MonthPicker

### DISCUSSION THREAD

**Discussion by sjur-gr** [navikt/Designsystemet#1893] [Komponenter]
🔗 [Link til DatePicker i Aksel](https://aksel.nav.no/komponenter/core/datepicker)
🔗 [Link til MonthPicker i Aksel](https://aksel.nav.no/komponenter/core/monthpicker)

Vi ønsker å vurdere hva som kan forbedre brukeropplevelsen av DatePicker og MonthPicker for utviklere og designere. Har du innspill til noe som vil gjøre disse komponenten bedre og mer brukervennlig for deg og brukerne av deres løsninger?

**Comment by KenAJoh** [comment]
Endringsforslag
- Egne inputs for RangedDatepicker.
- Fjerne `UNSAFE_`-prefix
- Bedre onValidation, dateChange-callbacks
